app/views/admin/_admin_options.html.erb

```diff
@@ -1,16 +1,42 @@
 <!-- BEGIN admin options -->
-  <div class="admin">
-	<h3 class="landmark heading">Admin Actions</h3>
+<div class="admin">
+  <% item_type_name = item.class == ExternalWork ? "External Work" :
```

Comment:
I suppose this "External Work" string should be wrapped into a ts() so we can round it up at some point in the future when we're ready to localise things